boards/FSM/Src/main.c

```diff
@@ -160,10 +167,23 @@ int main(void)
     MX_IWDG_Init();
     MX_ADC2_Init();
     MX_TIM4_Init();
+    MX_TIM16_Init();
+    MX_TIM17_Init();
     /* USER CODE BEGIN 2 */
     __HAL_DBGMCU_FREEZE_IWDG();
     Io_SharedHardFaultHandler_Init();
 
+    Io_FlowMeters_Init(&htim4);
+    primary_flow_meter = App_FlowMeter_Create(Io_FlowMeters_GetPrimaryFlowRate);
+    secondary_flow_meter =
+        App_FlowMeter_Create(Io_FlowMeters_GetSecondaryFlowRate);
+
+    Io_WheelSpeedSensors_Init(&htim16, &htim17);
+    left_wheel_speed_sensor =
+        App_WheelSpeedSensor_Create(Io_WheelSpeedSensors_GetLeftSpeed);
+    right_wheel_speed_sensor =
+        App_WheelSpeedSensor_Create(Io_WheelSpeedSensors_GetRightSpeed);
+
     Io_FlowMeters_Init(&htim4);
```

Comment:
`Io_FlowMeters_Init` is called twice??

Reply:
whoops this may have happened after rebasing. thanks.